specs/mention_spec.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert')
+const Botmock = require('botkit-mock')
+const mentionController = require('../features/mention.js')
+const responsesOnMention = require('../features/mention/replies')
```

Comment:
@nikitasakov I see `replies` is more suitable name here because you include replies.js file and you also use `replies` variable in the source code